modules/installation-required-vcenter-account-privileges.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-restricted-networks-vsphere.adoc
+// * installing/installing_vsphere/installing-vsphere.adoc
+// * installing/installing_vsphere/installing-vsphere-network-customizations.adoc
+
+[id="installation-required-vcenter-account-privileges_{context}"]
+= Required vCenter account privileges
+
+To install an {product-title} cluster in vCenter, the cluster requires access to an account with privileges to read and create the required resources. Using an account that has administrative privileges is the simplest way to access all of the necessary permissions.
+
+A user requires the following privileges to install an {product-title} cluster:
+
+* Datastore
+** *Allocate space*
+* Folder
+** *Create folder*
+** *Delete folder*
+* vSphere Tagging
+** All privileges
```

Comment:
Should this also be \*All privileges\*? Or is this a short hand for all listed privileges?

Reply:
And if so, should it be _all listed privileges_ or something so it's clear that it refers to everything there. Or maybe it not being bold signifies that it isn't a UI element. If someone has the UI visible, it probably makes sense, but just reading it, I wasn't sure.

Reply:
The bold signifies specific permissions that must be selected. We're using _All privileges_ as a general term for all permissions available for that specific component.